src/components/molecules/Avatar/props.ts

```diff
@@ -0,0 +1,7 @@
+import { HTMLAttributes } from 'react';
+
+export type Props = HTMLAttributes<HTMLDivElement> & {
+  url?: string | undefined;
+  alt?: string | undefined;
```

Comment:
They are already unioned by undefined because of the "optional(?)" flag. Leave just a "string".